### ISSUE TITLE
Add contextual information about web services

### ISSUE THREAD

**Issue by karenmajewicz** [geoblacklight/geoblacklight#831]
Some users may not understand what is being displaying in the map preview box - particularly the different types of geospatial web services or the index maps.

Two instances have added some info text:

- CUGIR has a description of index maps that appears under the download button: https://cugir.library.cornell.edu/catalog/cugir-009002

- Big Ten has added rollover help text at the top of the web services preview: https://geo.btaa.org/catalog/99455cda44a448188732de6161e28187

---------------
Could some version of these solutions be added to GeoBlacklight core. It should be customizable in terms of text and links.

**Comment by mejackreed** [comment]
💯 I would love it if this could be added to core.

**Comment by karenmajewicz** [comment]
addressed by PR #838
